collection-scripts/gather_vsphere_config

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+VSPHERE_CONFIG_EVENTS_PATH=$BASE_COLLECTION_PATH/vsphere_config_events
+
+if [ -z "$(oc get deployment -n openshift-cluster-storage-operator vsphere-problem-detector-operator | grep vsphere-problem-detector-operator)" ]; then
+exit
+fi
+
+METRICS="vsphere_vcenter_info vsphere_features vsphere_cluster_check_total vsphere_cluster_check_errors vsphere_node_check_errors vsphere_node_check_total vsphere_node_hw_version_total"
+mkdir -p $VSPHERE_CONFIG_EVENTS_PATH
+for METRIC in $METRICS; do
+/usr/bin/oc exec -c prometheus -n openshift-monitoring prometheus-k8s-0 -- curl -s --data-urlencode "query=$METRIC" http://localhost:9090/api/v1/query > $VSPHERE_CONFIG_EVENTS_PATH/$METRIC.json
```

Comment:
If we use oc inspect we already collect metrics for the operator. 

https://github.com/openshift/oc/blob/master/pkg/cli/admin/inspect/pod.go#L109

Is this a duplicate collection on the OpenShift Storage Operator?

Reply:
I was not aware and that `oc adm inspect` collected metrics.  If it does(I couldn't find it in a must-gather I took yesterday but I could have missed it) this would make this PR unnecessary.

Reply:
@rvanderp3 it gathers metrics on 'pods' - https://github.com/openshift/oc/blob/6352e607f262df44964a63004f85d81366accbbc/pkg/cli/admin/inspect/pod.go#L183-L196 so .. depending on the 'pod' your asking information from (and if you expose a `/metrics` endpoint) you get metrics for that pod. 

In short, going to the central metrics store (as you are here) may be breaking isolation (just a bit), however that dependent on how the metrics end up where they end up. 

You might need to check the Prometheus pods (data) to see the metrics (if they are collected). They may not be because were not exposing a query.